pkg/builder/webhook.go

```diff
@@ -84,8 +84,9 @@ func (blder *WebhookBuilder) WithLogConstructor(logConstructor func(base logr.Lo
 }
 
 // RecoverPanic indicates whether panics caused by the webhook should be recovered.
-func (blder *WebhookBuilder) RecoverPanic() *WebhookBuilder {
-	blder.recoverPanic = true
+// Defaults to true.
+func (blder *WebhookBuilder) RecoverPanic(recoverPanic *bool) *WebhookBuilder {
```

Comment:
Why take a bool pointer and not a value?

Reply:
Also sort of tried to be consistent with the  RecoverPanic *bool option we have for controllres, but I'm aware that we don't need that for these funcs :)

Reply:
Fixed